lib/constants.go

```diff
@@ -43,6 +43,10 @@ const (
 	MessagesToFetchPerInboxCall = 10000
 )
 
+// DisableNetworkManagerRoutines is used to disable the network manager routines.
+// This is useful for testing.
+var DisableNetworkManagerRoutines = false
+
```

Comment:
Global flags used testing are more likely to become tech debt in the long run that outweights the convenience in the short term. Is there a way you can still test without needing a global flag to disable the routines? Try defining a boolean constructor param on NetworkManager that disables the routines when flipped.

Reply:
Okay, agreed. Just fixed it.